app/Resources/views/page.html.twig

```diff
@@ -38,13 +38,24 @@
             {% set canonical = path(app.request.attributes.get('_route'), app.request.attributes.get('_route_params')) %}
         {% endif %}
 
+        {% block dublin_core %}
+
+            {% if item is defined and item.identifier is defined %}
+
+                <meta name="dc.identifier" content="{{ item.identifier }}">
+                <meta name="dc.relation.ispartof" content="elifesciences.org">
```

Comment:
Think it's ok to leave the same for all environments?

Reply:
This is fine. As the production hypothesis authority will only be implemented in one environment.

Reply:
Should we note the reason why `dc.identifier` and `dc.relation.ispartof` is lowercase here since it is not consistent with how we declare `DC.identifier` on the article page?

Reply:
Currently working on normalising them.